projects/BFDR.Tests/FetalDeathRecord_Should.cs

```diff
@@ -896,8 +896,56 @@ public void TestSetDeliveryLocation()
       Assert.Equal("Bob's Medical Center", br.BirthFacilityName);
       Assert.Equal("Abignale Hospital", br.FacilityMotherTransferredFrom);
     }
+    
+    [Fact]
+    public void SetOccupationIndustry()
+    {
+      FetalDeathRecord fetalDeathRecord = new FetalDeathRecord();
+      fetalDeathRecord.MotherOccupation = "Carpenter";
+      Assert.Equal("Carpenter", fetalDeathRecord.MotherOccupation);
+      fetalDeathRecord.MotherIndustry = "Construction";
+      Assert.Equal("Construction", fetalDeathRecord.MotherIndustry);
+      fetalDeathRecord.FatherOccupation = "Lawyer";
+      Assert.Equal("Lawyer", fetalDeathRecord.FatherOccupation);
+      fetalDeathRecord.FatherIndustry = "Legal Services";
+      Assert.Equal("Legal Services", fetalDeathRecord.FatherIndustry);
+
+      //ije translations
+      IJEFetalDeath ije = new IJEFetalDeath(fetalDeathRecord);
+      Assert.Equal("Carpenter", ije.MOM_OC_T.Trim());
+      Assert.Equal("Construction", ije.MOM_IN_T.Trim());
+      Assert.Equal("Lawyer", ije.DAD_OC_T.Trim());
+      Assert.Equal("Legal Services", ije.DAD_IN_T.Trim());
+      FetalDeathRecord fetalDeathRecord2 = ije.ToFetalDeathRecord();
+      Assert.Equal("Carpenter", fetalDeathRecord2.MotherOccupation);
+      Assert.Equal("Construction", fetalDeathRecord2.MotherIndustry);
+      Assert.Equal("Lawyer", fetalDeathRecord2.FatherOccupation);
+      Assert.Equal("Legal Services", fetalDeathRecord2.FatherIndustry);
+    }
 
     [Fact]
+    public void GetOccupationIndustry()
+    {
+       //parse
+      FetalDeathRecord record = new FetalDeathRecord(File.ReadAllText(TestHelpers.FixturePath("fixtures/json/FetalDeathReport.json")));
+      Assert.Equal("Secretary", record.MotherOccupation);
+      Assert.Equal("State Agency", record.MotherIndustry);
+      Assert.Equal("Teaching Assistant", record.FatherOccupation);
+      Assert.Equal("Elementary and Secondary Schools", record.FatherIndustry);
+      //set after parse
+      record.MotherOccupation = "Lawyer";
+      record.MotherIndustry = "Legal Services";
+      Assert.Equal("Lawyer", record.MotherOccupation);
+      Assert.Equal("Legal Services", record.MotherIndustry);
+      // convert to IJE
+      IJEFetalDeath ije = new(record);
```

Comment:
set the optional param to false to disable the validation, would look like new(record, false)